JS/Game.js

```diff
@@ -1,14 +1,39 @@
 class Game {
   constructor() {
     this.round = 0,
-    this.players = {}, // {player1: 'whatever is put in imput', ...}
+    this.players = [], // {player1: 'whatever is put in imput', ...}
     this.round3 = false,
     this.winner = null
+    this.data;
   }
 
   init() {
     //starts game
-    //this.round++
+    domUpdates.displayNames()
+    let playerOneName = $('.js-player1-name-input').val()
+    let playerTwoName = $('.js-player2-name-input').val()
+    let playerThreeName = $('.js-player3-name-input').val()
+    let playerOne = new Player(playerOneName);
+    this.players.push(playerOne)
+    let playerTwo = new Player(playerTwoName);
+    this.players.push(playerTwo)
+    let playerThree = new Player(playerThreeName);
+    this.players.push(playerThree)
```

Comment:
This looks like a good use-case for a `for` loop. Run the for loop three times and you can cut this down to two lines of code. Something like:

```
let player = $(`.js-player${i}-name-input`).val();
this.players.push(player);
```
```